src/watchlists/WatchList.jsx

```diff
@@ -30,15 +41,25 @@ export default function watchlist({ item }) {
           />
         </TouchableHighlight>
       </View>
-      {isOpen ? item.stocks.map((stock) => (
-        <WatchListItem
-          key={stock.id}
-          ticker={stock.ticker}
-          currPrice={stock.currPrice}
-          dailyChange={stock.dailyChange}
-          volumn={stock.volumn}
-        />
-      )) : null}
+      {isOpen && (
+        <>
+          {item.stocks.map((stock, i) => (
+            <WatchListItem
+              key={stock.id}
+              isEdit={isEdit}
+              isChecked={checkedItems[i]}
+              onSelect={() => updateChecked(i)}
+              ticker={stock.ticker}
+              currPrice={stock.currPrice}
+              dailyChange={stock.dailyChange}
+              volumn={stock.volumn}
+            />
+          ))}
+          {isEdit
+            ? <EditorMode checkedItems={checkedItems} isEdit={isEdit} toggle={setIsEdit} />
```

Comment:
I think these two could go to the same EditorFooter component
 - it renders the edit button if it not in edit mode
 - it renders the editor controls if it is in edit mode